.circleci/config.yml

```diff
@@ -56,7 +56,6 @@ commands:
               --set prisonApiBaseUrl="${PRISON_API_BASE_URL}" \
```

Comment:
@Eli-TW you can get rid of a lot of these 👍

Reply:
A lot of these gotten rid of in 96be84facb40eb3bbdd6084228930f9a0066fc5a